main/zoe/api/zoeGoverningContractFacet.md

```diff
@@ -0,0 +1,157 @@
+# Zoe Governing Contract Facet
```

Comment:
We've been moving from "Governing Contract" to just "Contract"

Reply:
ok, will update